model/src/main/java/de/danoeh/antennapod/model/feed/FeedItem.java

```diff
@@ -95,6 +108,13 @@ public FeedItem(long id, String title, String link, Date pubDate, String payment
         this.itemIdentifier = itemIdentifier;
         this.autoDownload = autoDownload;
         this.podcastIndexChapterUrl = podcastIndexChapterUrl;
+        if (transcriptUrl != null) {
+            this.podcastIndexTranscriptUrls = new Hashtable<String, String>();
+            this.podcastIndexTranscriptUrls.put(transcriptType, transcriptUrl);
+            this.hasTranscript = true;
+        }
+        // TT TODO  Load this from disk
+        // this.podcastIndexTranscriptText = transcriptText;
```

Comment:
Please remove TODO. Things like this tend to be forgotten and then stay in the code forever